### DISCUSSION TITLE
Can you play playlists with this?

### DISCUSSION THREAD

**Discussion by MatejaSrejic** [browniebroke/deezer-python#276] [Q&A] [answered]
I want to make a speaker with a Raspberry Pi that plays a certain playlist, can I do that with this library or will I have to use Selenium for e.g.?

**Comment by browniebroke** [comment, marked as answer]
I don't think the content of the tracks are exposed via the API, probably for DRM reasons. You can get [a track preview](https://developers.deezer.com/api/track), but that's just the first 30 seconds of it.
